samples/machine-learning/half-moons/README.md

```diff
@@ -18,6 +18,7 @@ The model is trained on a half-moons dataset, loaded in C# using the System.Text
 ## Prerequisites
 
 - The Microsoft [Quantum Development Kit](https://docs.microsoft.com/quantum/install-guide/).
+- To run the Python version you also will need to install the Python libraries `numpy` and `matplotlib`. You can install them from the command line by typing `pip install <library name>`.
```

Comment:
This is a good point; the samples repo contains an environment.yml file that should work for all Python-based samples, but that gets lost when onboarding a particular sample to docs.microsoft.com/samples. It may be worth making a copy of that environment.yml file here as well to automate installing NumPy and Matplotlib. I'm a little wary of telling people to `pip install` if our golden path for Python installation is to use `conda`.

Reply:
Do you mean to copy [this file](https://github.com/microsoft/Quantum/blob/geduardo/adding-prerequisites/samples/interoperability/python/environment.yml)? 

Another option could be to add Numpy and Matplotlib to the default environment created here: https://github.com/microsoft/Quantum/blob/geduardo/adding-prerequisites/samples/interoperability/python/environment.yml

Reply:
The environment.yml for samples/interoperability/python is fairly specialized to the process tomography case shown there, such that the environment.yml file can be a lot simpler in cases like this one, such as the one you added with a1656dc7a7bd0331e2064f009c191bd8eb7f1d47.